src/main.cpp

```diff
@@ -38,14 +39,18 @@ struct options {
 
 static void help(FILE *fp)
 {
-    fprintf(fp,
-            "%s [OPTIONS...]\n\n"
-            "  -c --conf-file                .conf file with configurations for "
-            "camera-streaming-daemon.\n"
-            "  -d --conf-dir <dir>          Directory where to look for .conf files overriding\n"
-            "                               default conf file.\n"
-            "  -h --help                    Print this message\n",
-            program_invocation_short_name);
+    fprintf(
+        fp,
+        "%s [OPTIONS...]\n\n"
+        "  -c --conf-file                   .conf file with configurations for "
+        "camera-streaming-daemon.\n"
+        "  -d --conf-dir <dir>              Directory where to look for .conf files overriding\n"
+        "                                   default conf file.\n"
+        "  -g --debug-log-level <level>     Set debug log level. Levels are\n"
+        "                                   <error|warning|notice|info|debug>\n"
+        "  -v --verbose                     Verbose. Same as --debug-log-level=debug\n"
+        "  -h --help                        Print this message\n",
+        program_invocation_short_name);
```

Comment:
This is not unrelated changed. And I didn't see that the check-coding has messed up with code. I'll fix the style